README.md

```diff
@@ -20,16 +20,13 @@ This project uses Swift Package Manager to pull in the frameworks from [axe-devt
 
 ### For Manual Testing
 
-1. Add your Deque API Key to `Login.swift`.
-_This app is ready for manual testing out of the box; running it in Xcode will show the floating action button._
+1. To manually test this app, use the [Mobile Analyzer](https://github.com/dequelabs/axe-devtools-mobile-analyzer).
 
 ### For Automated Testing
 
 1. Add your Deque API Key to `Login.swift`.
 
-1. In `SceneDelegate.swift`, comment out `axe?.showA11yFAB()` to remove manual mode, and uncomment the line for automated testing `axe?.showA11yFAB(customFAB: AutomatedTestFAB())`.
-
-1. Once you have a device or simulator ready to go, open `SampleUITests-UIKit.swift`, and tap on the diamond to run this suite. The simulator will launch, and the tests will run.
+2. Once you have a device or simulator ready to go, open `SampleUITests-XCUI.swift`, and tap on the diamond to run this suite. The simulator will launch, and the tests will run.
```

Comment:
I think the file needs to be renamed in this branch as well. Could also remove `-XCUI` all together since there is only one version now.

Reply:
@atristannewman  ^ still needs to be addressed. 
can either remove the rename, or add the renamed file to this PR

Reply:
Good call